Feat/rl 8386716812/update linked fellows

RyanCrux opened the pull request:
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

Update callout fellow sections to link to new pages. Update carousel logic to also point to new .md file template pages.

## Related Tickets & Documents
- Monday Tasks:
I've combined the following three Monday ticket into on branch. The tasks are for individual sections of the fellows site but they are similar tasks.
https://blackberg-group.monday.com/boards/1931719392/pulses/8386707988
https://blackberg-group.monday.com/boards/1931719392/pulses/8386716812
https://blackberg-group.monday.com/boards/1931719392/pulses/8386807722

## QA Instructions, Screenshots, Recordings

Review the links of the callout sections (2 of 3 pages). Review the carousel links. All links should be directed to the new .md templated files (fellows/fellow-name).

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

RyanCrux commented:
Ready for review:

Index:
https://www.dev-innovation.bgsandbox.com/ecosystem/views/programs/fellowship/index.html

Senior Innovation Fellowship:
https://www.dev-innovation.bgsandbox.com/ecosystem/views/programs/fellowship/senior-innovation-fellowship/

Entrepreneur Residence Fellowship:
https://www.dev-innovation.bgsandbox.com/ecosystem/views/programs/fellowship/entrepreneur-residence-fellowship/

ecstech-admin reviewed (changes requested):
Images paths broken. Correct asset paths for following (update to ../../assets)

- 	Official websites and Secure .gov icons 
- 	Close icon for VCL popup
- 	back to top icon
- 	footer va-color-seal-black-text.png
- 	(can update paths for footer images as they are pulling from ecosystem versus fellowship)
![image](https://github.com/user-attachments/assets/21f17bb0-8914-42ea-a8e0-44468d7e387d)


Main Nav, update link to National Centers home page (/ecosystem/views/national-centers/ rm "nci2i.html")
![image](https://github.com/user-attachments/assets/dfc0dcda-a70b-4cb6-9dcc-993b4a784633)


HTML being rendered as text (ex: https://dev-innovation.bgsandbox.com/ecosystem/views/programs/fellowship/fellow/colleen-mcquown/)
![image](https://github.com/user-attachments/assets/41b64e36-6869-4b04-9ed0-09121b1100e0)


Copy formatting not retained (Shull "About" example supposed to be 2 paragraphs although Kapur & Tran "About" did retain 2 paragraph formatting)
![image](https://github.com/user-attachments/assets/dc3e31f7-c1fd-4775-a847-047da58dfb72)


Carousel items, "Their Work", not uniform height on profile pages
![image](https://github.com/user-attachments/assets/d617dde3-06ce-4075-a13b-19d22aace3e0)


Significant size reduction (over 50%) converting jpg to webp (headshots and news images)

rb-blackberg reviewed (commented):
Close over all. Couple more items:

I still see the one paragraph in the About section like Jim mentioned - are we able to get line breaks in there?

Cards are still unequal height as well.

ecstech-admin reviewed (changes requested):
Looking good.

Other than the Search icon (broken), the broken image paths have been corrected but are linking to /ohil/assets rather than /ecosystem/views/fellowship/assets

Upon looking at _layout, I noticed for the profile pages you're using header.html and footer.html from _include. These includes are for the home page and probably why you were having problems with the paths. You should create and use new header/footer _include pages with the correct paths specific for the profile pages (ie: header-sub-sub.html & footer-sub-sub.html since the profile pages are 2 dirs below Fellowship home). Then you'll need to check/correct the paths for the other _includes (can probably reference paths from another repo)

Home page: "Current Fellows" cards do not have uniform height
![image](https://github.com/user-attachments/assets/d1659254-75bb-4f6d-a7a9-9d6acffb7271)


SIF/ERF pages: "Alumni" cards do not have uniform height
![image](https://github.com/user-attachments/assets/9de9b2c9-e179-46d8-ae4b-3224fded0d5d)


Finally, somehow the Home "News & Events" carousel card heights got affected (no longer uniform height)
![image](https://github.com/user-attachments/assets/98e0d734-402a-45ae-9367-518f44ca1c7d)

RyanCrux commented:
@ecstech-admin Card height issue for the other sections was all related to one fix and is sorted. I updated the search icon path on the main header and adjusted the layout page to use sub header/footer includes. The image paths are relative to the fellowship folder structure. Everything is loading correctly now but after review, if you think something should be structured differently, lets hop on a call to make sure we are on the same page. Thanks for the effort in this review!

ecstech-admin reviewed (changes requested):
Just about there...

Search icon path on SIF and ERF is broken.
Official websites and Secure .gov icon paths broken on profile pages.

ecstech-admin reviewed (approved):
👍

🚀
